modules/integration/src/test/scala/scala/cli/integration/InstallAndUninstallCompletionsTests.scala

```diff
@@ -0,0 +1,37 @@
+package scala.cli.integration
+
+import com.eed3si9n.expecty.Expecty.expect
+
+import scala.util.Properties
+
+class InstallAndUninstallCompletionsTests extends munit.FunSuite {
+  val zshRcFile  = ".zshrc"
+  val bashRcFile = ".bashrc"
+  val testInputs = TestInputs(Seq(os.rel / zshRcFile -> "", os.rel / bashRcFile -> ""))
+
+  def runInstallAndUninstallCompletions(): Unit = {
+    testInputs.fromRoot { root =>
+      val zshRcPath  = root / zshRcFile
+      val bashRcPath = root / bashRcFile
+      // install completions to the dummy rc files
+      os.proc(TestUtil.cli, "install-completions", "--rc-file", zshRcPath, "--shell", "zsh").call(
+        cwd = root
+      )
+      os.proc(TestUtil.cli, "install-completions", "--rc-file", bashRcPath, "--shell", "bash").call(
+        cwd = root
+      )
+      expect(!os.read.lines(zshRcPath).isEmpty)
+      expect(!os.read.lines(bashRcPath).isEmpty)
+      // uninstall completions from the dummy rc files
+      os.proc(TestUtil.cli, "uninstall-completions", "--rc-file", zshRcPath).call(cwd = root)
+      os.proc(TestUtil.cli, "uninstall-completions", "--rc-file", bashRcPath).call(cwd = root)
+      expect(os.read.lines(zshRcPath).isEmpty)
```

Comment:
similarly, instead of checking if it's empty, just compare if it's the same contents as it was before the changes